src/Device/Driver.hpp

```diff
@@ -101,6 +101,24 @@ class Device {
   virtual bool PutBallast(double fraction, double overload,
                           OperationEnvironment &env) = 0;
 
+  /**
+    * Send the new crew weight value to the device.
+    *
+    * @param total weight in kg of all crew members
+    * @return true on success
+    */
+  virtual bool PutCrewWeight(double weight,
+                   OperationEnvironment &env) = 0;
+
+  /**
+      * Send the new empty mass value of the glider to the device.
+      *
+      * @param empty mass in kg of the glider
+      * @return true on success
+      */
+   virtual bool PutEmptyMass(double weight,
```

Comment:
PutEmpty**Mass** but the parameter is called **weight**??

Reply:
will be renamed in plane masses PR